src/board/v3/nap/nap_common.c

```diff
@@ -119,3 +124,55 @@ static void nap_exti_thread(void *arg)
   }
 }
 
+u8 nap_conf_rd_version_string(char version_string[])
+{
+  u8 i = 0;
+  u32 reg = 0;
+
+  do {
+    NAP->CONTROL = (i << 1);
```

Comment:
Yes, two other bits. One to select the raw sample grabber input and the other for selecting external event edge sensitivity. I assumed we would only read out the version at startup and the DNA during device bringup so it wouldn't matter if you overwrite anything. But probably better to play it safe. I'll change that.